source/documentation/errors.html.md.erb

```diff
@@ -86,191 +86,229 @@ All errors from TGP Digital will return a JSON map with two keys: "code" and "me
     <td>Mandatory field category was missing from body or is in the wrong format</td>
   </tr>
   <tr>
     <td>15</td>
     <td>assessments[].assessmentId</td>
+    <td>Create TGP record, Update TGP record</td>
     <td>Optional field assessmentId is in the wrong format</td>
   </tr>
   <tr>
     <td>16</td>
     <td>assessments[].primaryCategory</td>
+    <td>Create TGP record, Update TGP record</td>
     <td>Optional field primaryCategory is in the wrong format</td>
   </tr>
   <tr>
     <td>17</td>
     <td>assessments[].primaryCategory.condition.type</td>
+    <td>Create TGP record, Update TGP record</td>
     <td>Optional field type is in the wrong format</td>
   </tr>
   <tr>
     <td>18</td>
     <td>assessments[].primaryCategory.condition.conditionId</td>
+    <td>Create TGP record, Update TGP record</td>
     <td>Optional field conditionId is in the wrong format</td>
   </tr>
   <tr>
     <td>19</td>
     <td>assessments[].primaryCategory.condition.conditionDescription</td>
+    <td>Create TGP record, Update TGP record</td>
     <td>Optional field conditionDescription is in the wrong format</td>
   </tr>
   <tr>
     <td>20</td>
     <td>assessments[].primaryCategory.condition.conditionTraderText</td>
+    <td>Create TGP record, Update TGP record</td>
     <td>Optional field conditionTraderText is in the wrong format</td>
   </tr>
   <tr>
     <td>21</td>
     <td>supplementaryUnit</td>
+    <td>Create TGP record, Update TGP record</td>
     <td>Optional field supplementaryUnit is in the wrong format</td>
   </tr>
   <tr>
     <td>22</td>
     <td>measurementUnit</td>
+    <td>Create TGP record, Update TGP record</td>
     <td>Optional field measurementUnit is in the wrong format</td>
   </tr>
   <tr>
     <td>23</td>
     <td>comcodeEffectiveFromDate</td>
+    <td>Create TGP record, Update TGP record</td>
     <td>Mandatory field comcodeEffectiveFromDate was missing from body or is in the wrong format</td>
   </tr>
   <tr>
     <td>24</td>
     <td>comcodeEffectiveToDate</td>
+    <td>Create TGP record, Update TGP record</td>
     <td>Optional field comcodeEffectiveToDate is in the wrong format</td>
   </tr>
   <tr>
     <td>25</td>
     <td>recordId</td>
+    <td>Update TGP record, Get TGP record, Remove TGP record</td>
     <td>The recordId has been provided in the wrong format</td>
   </tr>
   <tr>
     <td>26</td>
     <td>recordId</td>
+    <td>Update TGP record, Get TGP record, Remove TGP record</td>
     <td>The requested recordId to update doesn’t exist</td>
   </tr>
   <tr>
     <td>27</td>
     <td>adviceStatus</td>
+    <td>Update TGP record, Get TGP record</td>
     <td>There is an ongoing advice request and the record can not be updated</td>
   </tr>
   <tr>
     <td>28</td>
     <td>lastUpdatedDate</td>
+    <td>Get TGP record, Remove TGP record</td>
     <td>The URL parameter lastUpdatedDate is in the wrong format</td>
   </tr>
   <tr>
     <td>29</td>
     <td>page</td>
+    <td>Get TGP record, Remove TGP record</td>
```

Comment:
This was a mistake in the documentation and should just be Get - I've updated the Confluence now